src/Orc.SupportPackage/Orc.SupportPackage.Shared/Services/Interfaces/ISupportPackageService.cs

```diff
@@ -11,6 +11,6 @@ namespace Orc.SupportPackage
 
     public interface ISupportPackageService
     {
-        Task<bool> CreateSupportPackageAsync(string zipFileName);
+        Task<bool> CreateSupportPackageAsync(string zipFileName, string[] excludeFileNamePatterns = null);
```

Comment:
This is a breaking change. Please introduce a *new* method and mark this one with obsolete.